### ISSUE TITLE
[Bug][A11Y][Checkbox] Incorrect accessibility label

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#514]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 14+

### OS Version

iOS26

### App version

0.6.0 (520)

### Describe the issue

The accessibility label is read by Voice Over but is also used by Voice Control, so users can control the app by voice only, for example by saying the visible label, for example "touch Label".

Here, the label is "CheckboxLabel" concatenation of "checkbox" (role) and label "Label"

<img width="471" height="271" alt="" src="https://github.com/user-attachments/assets/7487ea16-ced2-4bca-be79-55b124af1d53" />

### Expected Behavior

Label should contain only the label of the checkbox
The role should be in trait (for basics traits) or value (for custom role)

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr) to get access.

Display name: DesignToolbox (alpha) (589)
Version: 0.7.0
Build type: Alpha
Build details (GitHub): 522
cc @B3nz01d (product owner)
cc @nouha06 @AhmedAmineZr (dev team)
cc @pya35  (accessibility team)

**Comment by nouha06** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr) to get access.

Display name: Design Toolbox Flutter (589)
Version: 0.7.0
Build type: Alpha
Build details (GitHub): 522
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (Accessibility)

**Comment by pya35** [comment]
@nouha06 no more role on checkbox, it's not possible anymore to interact with the chekbox using Voice Control or Switch Control 

<img width="480" height="640" alt="" src="https://github.com/user-attachments/assets/4483c7ce-daa8-4d6b-ab84-3d7104d5fa6e" />

<img width="297" height="205" alt="" src="https://github.com/user-attachments/assets/8ac4aa47-b898-4c93-8fd3-bfec8e271494" />

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr) to get access.

Display name: DesignToolbox (alpha) (596)
Version: 0.7.0
Build type: Alpha
Build details (GitHub): 522
cc @B3nz01d (product owner)
cc @nouha06 @AhmedAmineZr (dev team)
cc @pya35 (accessibility team)

**Comment by nouha06** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr) to get access.

Display name: Design Toolbox Flutter (596)
Version: 0.7.0
Build type: Alpha
Build details (GitHub): 522
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (Accessibility)

**Comment by pya35** [comment]
@nouha06 reproduced on 0.7.0 (595), instead of "Label, the current accessible label is "CheckboxLabel" 

<img width="471" height="271" alt="" src="https://github.com/user-attachments/assets/6dfaf539-105d-4f82-81a5-fe7ebc246879" />

**Comment by nouha06** [comment]
📣 New TestFlight Alpha upload available 🚀 (iOS)

Please, get the new build from TestFlight app, or contact the project maintainers (@AhmedAmineZr) to get access.

Display name: Design Toolbox Flutter (658)
Version: 1.2.0
Build type: Alpha
Build details (GitHub): 514
cc @B3nz01d (product owner)
cc @AhmedAmineZr @nouha06 (dev team)
cc @pya35 (Accessibility)

**Comment by nouha06** [comment]
📣 New Firebase App Distribution Alpha upload available 🚀 (Android)

Please, get the new build from App Tester app, or contact the project maintainers (@AhmedAmineZr) to get access.

Display name: DesignToolbox (alpha) (658)
Version: 1.2.0
Build type: Alpha
Build details (GitHub): 514
cc @B3nz01d (product owner)
cc @nouha06 @AhmedAmineZr (dev team)
cc @pya35 (accessibility team)

**Comment by pya35** [comment]
OK
